packages/roosterjs-content-model-api/lib/publicApi/image/insertImage.ts

```diff
@@ -41,6 +45,22 @@ function insertImageWithSrc(editor: IEditor, src: string) {
         },
         {
             apiName: 'insertImage',
+            onNodeCreated: (model, node) => {
```

Comment:
Actually, you can do:
```ts
editor.formatContentModel(model => {
 const image = createImage(...);
 mergeModel(...);
 image.isSelected = true;
 setSelection(model, image) // This will clear all other existing selections and only select this image
 
 return true;
});
```

Reply:
I see, sounds good. Changed to use this util.
Thanks